datajoint/base_relation.py

```diff
@@ -157,6 +157,39 @@ def _repr_helper(self):
         return "%s.%s()" % (self.__module__, self.__class__.__name__)
 
     # --------- SQL functionality --------- #
+    # TODO: finish this implementation
+    # @property
+    # def db_definition(self):
+    #     tdef = ...
+    #     if self.is_declared:
+    #         comment = self.table_comment
+    #         tdef = '# {comment}\n'.format(comment=comment) if comment else ''
+    #
+    #         #--------------------------
+    #         # TODO: remove this
+    #         from IPython import embed
+    #         embed()
+    #         exit()
+    #         #--------------------------
+    #
+    #         keys = ""
+    #         attributes = ""
+    #
+    #     return tdef
+
+    @property
+    def table_comment(self):
```

Comment:
We already parse `SHOW CREATE TABLE` in `dependencies.py` to get the foreign keys.  We also parse individual attributes in `heading.py` but there we do it from  `SHOW FULL COLUMNS` instead.  The only other thing we may need to get from the definitions are the indexes.  Now I think it's okay to keep things as you implemented them here.